### PR TITLE
updated old endpoints and added back the faker profiles

### PR DESCRIPTION
**Pull request by meloegel**
Updated the old model/end points with the new naming convention. Added back the faker data for profiles, may remove it later but are using it to see how the dummy data profiles apply to the FE.

### CODE REVIEW

**Review by anthonyj713 — APPROVED** [review]
Fixes problems by adding seed data and changing naming conventions.

**Review by derekdyer0309 — APPROVED** [review]

**Comment by derekdyer0309** [comment]
@LeesahMasko take a look and approve when you can and I will merge

File: api/bridge/bridgeModel.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const updateBridge = async (id, changes) => {
 
 const findbyStage = async (stage) => {
   console.log(stage);
```
Comment (Vippsi) [inline api/bridge/bridgeModel.js:26]: Console.log isn't nessicarry after development, please remove before production

File: data/seeds/001_profiles.js.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const faker = require('faker');
```
Comment (Vippsi) [inline data/seeds/001_profiles.js.js:1]: Consider adding a comment detailing what faker is

**Review by Vippsi — COMMENTED** [review]
Just remove the console log, everything else looks great!

**Pull request merged**